integreat_cms/cms/models/pages/page.py

```diff
@@ -216,14 +216,11 @@ def explicitly_archived_ancestors(self):
         """
         This returns all of the page's ancestors which are archived.
 
-        :return: The QuerySet of archived ancestors
+        :return: The list of archived ancestors
         :rtype: list [ ~integreat_cms.cms.models.pages.page.Page ]
         """
-        return [
-            ancestor
-            for ancestor in self.get_cached_ancestors()
-            if ancestor.explicitly_archived
-        ]
+
+        return self.region.explicitly_archived_ancestors_query(fixed=self)
```

Comment:
```suggestion
        return [
            ancestor
            for ancestor in self.get_cached_ancestors()
            if ancestor.explicitly_archived
        ]
```
Hmm, I'd like to revert this change since it bypasses the cached ancestors and might cause additional queries in some circumstances.